src/main/java/com/github/minecraftschurlimods/arsmagicalegacy/common/init/AMItems.java

```diff
@@ -134,12 +131,16 @@ public ItemStack makeIcon() {
     RegistryObject<ForgeSpawnEggItem>            LIFE_GUARDIAN_SPAWN_EGG      = ITEMS.register("life_guardian_spawn_egg", () -> new ForgeSpawnEggItem(AMEntities.LIFE_GUARDIAN, 0x12e780, 0xc9bc2f, ITEM_64));
     RegistryObject<ForgeSpawnEggItem>            ARCANE_GUARDIAN_SPAWN_EGG    = ITEMS.register("arcane_guardian_spawn_egg", () -> new ForgeSpawnEggItem(AMEntities.ARCANE_GUARDIAN, 0x7f3280, 0xc9bc2f, ITEM_64));
     RegistryObject<ForgeSpawnEggItem>            ENDER_GUARDIAN_SPAWN_EGG     = ITEMS.register("ender_guardian_spawn_egg", () -> new ForgeSpawnEggItem(AMEntities.ENDER_GUARDIAN, 0x000000, 0xc9bc2f, ITEM_64));
+    RegistryObject<ForgeSpawnEggItem>            DRYAD_SPAWN_EGG              = ITEMS.register("dryad_spawn_egg", () -> new ForgeSpawnEggItem(AMEntities.DRYAD, 0x166822, 0x683d16, ITEM_64));
+    RegistryObject<ForgeSpawnEggItem>            MAGE_SPAWN_EGG               = ITEMS.register("mage_spawn_egg", () -> new ForgeSpawnEggItem(AMEntities.MAGE, 0x777777, 0x7b1a7c, ITEM_64));
+    RegistryObject<ForgeSpawnEggItem>            MANA_CREEPER_SPAWN_EGG       = ITEMS.register("mana_creeper_spawn_egg", () -> new ForgeSpawnEggItem(AMEntities.MANA_CREEPER, 0x1abfb5, 0x368580, ITEM_64));
+//    RegistryObject<NatureScytheItem>             NATURE_SCYTHE                = ITEMS.register("nature_scythe", NatureScytheItem::new);
+//    RegistryObject<WintersGraspItem>             WINTERS_GRASP                = ITEMS.register("winters_grasp", WintersGraspItem::new);
```

Comment:
Why are these commented out?

Reply:
These are unique boss drops which are not in the scope of this PR; I implemented them for testing purposes only

Reply:
if they are implemented leave them in as WIP if not add a todo comment